tests/local/missing-features/rec-group.wast

```diff
@@ -2,9 +2,12 @@
   (module (rec))
   "requires `gc` proposal to be enabled")
 
-(assert_invalid
-  (module (rec (type (func))))
-  "requires `gc` proposal to be enabled")
+;; As a size optimization, we encode explicit rec groups of one type into the
+;; equivalent implicit rec group of just that type's direct encoding.
+;;
+;; (assert_invalid
+;;   (module (rec (type (func))))
+;;   "requires `gc` proposal to be enabled")
```

Comment:
This seems a result of the `wast` change above, but is a test I'd ideally like to keep. If the `wast` change can't be reverted then can this switch to using `module binary` instead of the text format?